outputs.tf

```diff
@@ -1,6 +1,7 @@
 
 locals {
-  template_version = "1_0_1" # '.' not allowed in labels
+  # major_minor_patch: major incremented on breaking changes, while patches are risk-free or important security fixes.
```

Comment:
Isn't this a risk-free update?

Reply:
:thinking: should be fairly safe, I can make it a patch.